_resources/2020-04-03-where-to-safely-shop-during-covid-1.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Where to Safely Shop during Covid-19
 category: Educational content
-country: United States
+country: USA
```

Comment:
This was causing an extra, redundant category

Reply:
Check these kinds of content changes with @rousik, as these files are generated automatically

Reply:
@mverzilli I removed the manual edit to this file.